config/buses/kafka/README.md

```diff
@@ -2,6 +2,12 @@
 
 Deployment steps:
 1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
+1. If you want to set the default Knative Bus to Kafka run the following command to edit the Knative Eventing configuration:
+    ```shell
+    kubectl get cm flow-controller-config -n knative-eventing -oyaml  \
+    | sed -e 's/stub/kafka/' \
```

Comment:
replacing `stub` is a bit aggressive as it will match other places in the document, how about changing the matching pattern to `default-cluster-bus: stub`?

Reply:
+1

Reply:
Good point - updated.